src/index.js

```diff
@@ -32,4 +32,9 @@ const startApp = () => {
   app.listen(port, () => logger.info(listening));
 };
 
-logger.initKafkaLoggingProducer().then(startApp);
+try {
+  logger.initKafkaLoggingProducer().then(startApp);
+} catch (err) {
+  logger.error(err);
```

Comment:
Do you have a test for this? Are you sure this shouldn't be `.then(startApp).catch((err) => ...)`?

Reply:
Fixed, and added test